src/components/WarningView.tsx

```diff
@@ -24,7 +24,7 @@ const WarningView = () => {
     if (fileWarningStrings.length <= 0) return null;
 
     return (
-        <div className="warning-view">
+        <div className="tw-w-full tw-max-w-5xl tw-self-center">
```

Comment:
Max width was max-width: 1016px;
tw-max-w-5xl might be not same value and there might be content overflow by couple of pixels

Reply:
Yes I am ware if this and I also took care of that